src/locales/sr_RS/person/name.ts

```diff
@@ -0,0 +1,3 @@
+export default [
+  { value: '{{person.first_name}} {{person.last_name}}', weight: 1 },
```

Comment:
```suggestion
  { value: '{{person.firstName}} {{person.lastName}}', weight: 1 },
```

Ref: https://github.com/faker-js/faker/blob/next/src/locales/en/person/name.ts#L2

Reply:
Done.